tests/lib/rules/no-restricted-imports.js

```diff
@@ -282,6 +282,15 @@ ruleTester.run("no-restricted-imports", rule, {
                     importNames: ["Foo"]
                 }]
             }]
+        },
+        {
+            code: "import Foo from '../../my/relative-module';",
+            options: [{
+                patterns: [{
+                    group: ["**/my/relative-module"],
+                    importNamePattern: "^Foo"
+                }]
+            }]
```

Comment:
Can we add the following valid test cases too:

```
import Foo from 'foo';
import {Bar as Foo } from "foo";
import * as Foo from "foo";
import { Bar } from "foo";
import { Bar } from "../../my/relative-module";

export *  from "foo";
export { Bar as Foo } from "foo";
export { Bar } from "foo";
```

Reply:
Done!